settings.py

```diff
@@ -70,7 +70,7 @@
 ES_DEFAULT_NUM_REPLICAS = 0
 
 SITE_URL = os.environ.get('OLYMPIA_SITE_URL') or 'http://localhost:8000'
-SERVICES_DOMAIN = 'localhost:8000'
+SERVICES_DOMAIN = SITE_URL.replace('http://', '').replace('https://', '')
 SERVICES_URL = 'http://%s' % SERVICES_DOMAIN
```

Comment:
Yeah, local dev only. but actually I think I can just replace this with `SERVICES_URL = SITE_URL` - we do use `SERVICES_DOMAIN` so I gotta keep it, but I don't see the point of reconstructing the url...